pom.xml

```diff
@@ -55,7 +56,7 @@
     </developers>
 
     <properties>
-        <revision>0.1.56</revision>
+        <revision>0.1.55.1</revision>
```

Comment:
@jvz’s #6 dropped the fourth version component on top of the upstream release version which indicates the Jenkins packaging system revision. I suspect this was an accident.